app/views/oidc/api_key_roles/show.html.erb

```diff
@@ -0,0 +1,50 @@
+<% @title = t(".api_key_role_name", name: @api_key_role.name) %>
+
+<div class="t-body">
+  <h3 class="t-list__heading"><%= OIDC::ApiKeyRole.human_attribute_name(:token) %></h3>
+  <div class="push--s">
+    <code><%= @api_key_role.token %></code>
+  </div>
+
+  <h3 class="t-list__heading"><%= OIDC::ApiKeyRole.human_attribute_name(:provider) %></h3>
+  <div class="push--s">
+    <%= link_to @api_key_role.provider.issuer, @api_key_role.provider.issuer %>
+  </div>
+
+  <h3 class="t-list__heading"><%= OIDC::ApiKeyRole.human_attribute_name(:api_key_permissions) %></h3>
+  <div class="push--s">
+  <code><pre><%= JSON.pretty_generate @api_key_role.api_key_permissions.as_json %></pre></code>
+  </div>
+
+  <h3 class="t-list__heading"><%= OIDC::ApiKeyRole.human_attribute_name(:access_policy) %></h3>
+  <div class="push--s">
+    <code><pre><%= JSON.pretty_generate @api_key_role.access_policy.as_json %></pre></code>
+  </div>
+
+  <h3 class="t-list__heading"><%= OIDC::ApiKeyRole.human_attribute_name(:id_tokens) %></h3>
+
+  <div class="push--s">
+    <table class="owners__table">
+      <thead>
+        <tr class="owners__row owners__header">
+          <td class="owners__cell"><%= OIDC::IdToken.human_attribute_name(:created_at) %></td>
+          <td class="owners__cell"><%= ApiKey.human_attribute_name(:expires_at) %></td>
+          <td class="owners__cell"><%= OIDC::IdToken.human_attribute_name(:jti) %></td>
+          <td class="owners__cell"><%= OIDC::IdToken.human_attribute_name(:claims) %></td>
+          <td class="owners__cell"><%= OIDC::IdToken.human_attribute_name(:header) %></td>
+        </tr>
+      </thead>
+      <tbody class="t-body">
+        <% @api_key_role.id_tokens.each do |id_token| %>
+          <tr class="owners__row <%= 'owners__row__invalid' if id_token.api_key.expired? %>">
+            <td class="owners__cell"><%= id_token.created_at %></td>
+            <td class="owners__cell"><%= id_token.api_key.expires_at %></td>
+            <td class="owners__cell"><code><%= id_token.jti %></code></td>
+            <td class="owners__cell"><code><%= id_token.claims.to_json %></code></td>
+            <td class="owners__cell"><code><%= id_token.header.to_json %></code></td>
+          </tr>
```

Comment:
This is pretty overwhelming. I think we can just get by with expired_at, created_at, and jti.